web/app/Controller/AddGameController.php

```diff
@@ -65,8 +66,39 @@ public function index(): void
         if ($id_black == $id_white) {
             redirect(url('calculator', Null, ['Status'=>'GleicherSpieler'])->getAbsoluteUrl());
         }
-
-
+        $sql = "SELECT username FROM person WHERE id = '$id_black' LIMIT 1";
```

Comment:
variablen in strings können zu unvorhersehbaren problemen führen. besser konkatenieren ( 'a' . $b . 'c' ).
insbesondere in sql queries sollten keine variablen stehen. vor allem dann nicht, wenn die eingabe vom benutzer kommt.